.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: php
-
+dist: trusty
```

Comment:
please instead remove hhvm from the `php` list and add another entry in matrix: include: that has php: hhvm and dist: trusty lines. that way we only change the dist for hhvm.